corehq/apps/registration/static/registration/js/register_new_user.js

```diff
@@ -3,10 +3,12 @@ $(function () {
     var initial_page_data = hqImport('hqwebapp/js/initial_page_data').get;
 
     // Demo CTA test
-    var kissmetrics = hqImport('analytix/js/kissmetrix');
-    kissmetrics.identifyTraits({'demo UI': (initial_page_data("demo_test") ? "on" : "off")});
-    $("#demo-test-button").click(function() {
-        kissmetrics.track.event("Get Demo button clicked");
+    hqImport("analytix/js/hubspot").then(function() {
+        var kissmetrics = hqImport('analytix/js/kissmetrix');
```

Comment:
What does it mean that the demo button requires hubspot? It looks like it shows up even before hubspot has been loaded?

Reply:
It's "required" on the html side: both demo buttons are inside of an `{% if ANALYTICS_IDS.HUBSPOT_API_ID %}`